Marble Blast Platinum/platinum/client/scripts/mp/commands.cs

```diff
@@ -121,6 +121,9 @@
 function clientCmdFoundEgg(%time, %eggName, %eggPickup) {
 	PlayGui.showEggTime(%time);
 
+	statsGetTopScoreModes(PlayMissionGui.getMissionInfo()); // For the egg on pause screen.
```

Comment:
Need to throw if (lb()) around this so you don't do web requests in offline mode